tortoise/queryset.py

```diff
@@ -372,6 +374,11 @@ def prefetch_related(self, *args: str) -> 'QuerySet':
                 queryset._prefetch_map[first_level_field].add(forwarded_prefetch)
         return queryset
 
+    def explain(self, format: Optional[str] = None, **options) -> 'QuerySet':
+        queryset = self._clone()
+        queryset._explain = True
+        return queryset
```

Comment:
As I mentioned, I would rather this be an `async def` and return the explain immediately, so as to minimise adding complexity into `QuerySet`

Reply:
Yup!